src/asciidoc/docs/blueprints/magazineCover.adoc

```diff
@@ -10,8 +10,7 @@ image::magazine-cover.png[Make a magazine cover template on Koji,role="img-overv
 
 === Prerequisites
 
-include::{includespath}/prereqs.adoc[tag=webdev]
-include::{includespath}/prereqs.adoc[tag=remixproc]
+include::{includespath}/prereqs.adoc[tag=webdev;remixproc]
```

Comment:
```suggestion
include::{includespath}/prereqs.adoc[tags=webdev;remixproc]
```
You have to use the plural for multiple tags to work. Please update on the rest of the pages, too. Be sure to run/preview the docs locally to help catch these syntax oddities before reviews.

Reply:
Fixed. Sorry about that.